container/podman/README.md

```diff
@@ -0,0 +1,16 @@
+# Podman test suite
+Verify that podman can run integration tests with the system. \
+Test Maintainer: [Major Hayden](mailto:major@redhat.com)
+
+## How to run it
+Please refer to the top-level README.md for common dependencies.
+
+### Install dependencies
+```bash
+dnf -y install $(grep '^dependencies=' metadata | awk -F'=' '{print $NF}' | tr ';' ' ')
```

Comment:
Nice! This is going to be the same for every test that uses the modern `metadata` stuff, right? I wonder if there is a better way to standardise how to install the dependencies. It sucks a bit to have to include these instructions everywhere 🙁

Reply:
Yes, will use `metadata` for all test cases. 
I'll file a script to install dependencies and update README.md, please wait for a while...

Reply:
It doesn't have to be in this PR BTW! I was just wondering, thinking out loud, and wondering about your opinion.

So we can merge as-is if you want, or I can wait for the script if you prefer.

Reply:
The script to install dependencies is added. Please take a look again, thanks!